File: static/include/examples-generated/data-query-examples.snippet.data-query-count.py
tabular_data_mql_count = await data_client.tabular_data_by_mql(
    organization_id=ORG_ID,
    query=[
        {
            "$match": {
                "component_name": "sensor-1"
            },
        }, {
            "$count": "count"
        }
    ]
)
print(f"Tabular Data: {tabular_data_mql_count}")

tabular_data_sql_count = await data_client.tabular_data_by_sql(
    organization_id=ORG_ID,
    query="SELECT count(*) FROM readings "
    "WHERE component_name = 'sensor-1'"
)
print(f"Tabular Data: {tabular_data_sql_count}")

File: static/include/examples-generated/data-query-examples.snippet.data-query-count.ts
const tabularDataMQLCount = await client.dataClient.tabularDataByMQL(
    ORG_ID,
    [
        { "$match": { "component_name": "sensor-1" } },
        { "$count": "count" }
    ]
);
console.log(tabularDataMQLCount);

const tabularDataSQLCount = await client.dataClient.tabularDataBySQL(
    ORG_ID,
    "SELECT count(*) FROM readings WHERE component_name = 'sensor-1'"
);
console.log(tabularDataSQLCount);
